internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -45,6 +46,12 @@ const (
 	maxIndexNameLength = 63
 )
 
+// specialCharacters are potential problematic characters that are replaced with `_`.
+// They are operators and special characters of pg and white space.
+// See https://www.postgresql.org/docs/15/sql-syntax-lexical.html#SQL-SYNTAX-OPERATORS
+// and https://www.postgresql.org/docs/15/sql-syntax-lexical.html#SQL-SYNTAX-SPECIAL-CHARS
+var specialCharacters = regexp.MustCompile("[-+*/<>=~!@#%^&|`?()\\[\\],;:. ]")
```

Comment:
I think negating is a safer option there. Something like `[^a-z0-9_]`, but a special check is needed for the first symbol that should be `[a-z]`

Reply:
That works as well. Let me do that.

Reply:
Okay so I think I can match with `[^a-z][^a-z0-9_]*`